app/scripts/modules/amazon/pipeline/stages/disableCluster/disableClusterExecutionDetails.html

```diff
@@ -9,7 +9,7 @@
                         <account-tag account="stage.context.credentials"></account-tag>
                     </dd>
                     <dt>Region</dt>
-                    <dd>{{stage.context.region}}</dd>
+                    <dd>{{stage.context.regions[0]}}</dd>
```

Comment:
should probably change to `{{stage.context.regions.join(', ')}}`, since the stage supports multiple regions